_includes/install-docker-for-mac.md

```diff
@@ -0,0 +1,8 @@
+### Install Docker for Mac
+
+If you haven't installed Docker for Mac, please follow the installation instructions on [Docker website][install-d4m].
+
+Once it's runing you should see <img alt="Docker Icon in the Mac OS menu bar" src="docker-for-mac-menu-bar-icon.png"
```

Comment:
No idea, will have to test this when we try to publish. I am suspecting gitdown doesn't do anything about images, will check now.